clients/da-vinci-client/src/main/java/com/linkedin/davinci/client/DaVinciConfig.java

```diff
@@ -99,15 +84,6 @@ public DaVinciConfig setStorageClass(StorageClass storageClass) {
     return this;
   }
 
-  public NonLocalAccessPolicy getNonLocalAccessPolicy() {
-    return nonLocalAccessPolicy;
-  }
-
-  public DaVinciConfig setNonLocalAccessPolicy(NonLocalAccessPolicy nonLocalAccessPolicy) {
-    this.nonLocalAccessPolicy = nonLocalAccessPolicy;
-    return this;
-  }
```

Comment:
Let's leave these two functions as is and mark them deprecated instead. We can leave the setter empty and the getter return `FAIL_FAST`.

Once we've gotten everyone to migrate off, we can remove it. Otherwise, it becomes a breaking change.

Reply:
We will bump the major version to force customer to get rid of the usage.